### PR TITLE
Publish to Dockerhub

### PR DESCRIPTION
**Pull request by k-nut**
Open items:
- [x] Update documentation
  - [x] How the docker setup works (link to jedeschule-api)
  - [x] How to trigger an update of all scrapers
  - [x] How to trigger an update for an individual scraper
- [x] Move crontab to the docker host instead of running from within docker
- [x] Clean up commit history

~~Currently awaiting #77 to be merged so that experimenting with the docker containers becomes faster since we don't need lxml anymore.~~ Done now ✅

### CODE REVIEW

File: scrape_all.sh [hunk 1]
```diff
@@ -0,0 +1,2 @@
+scrapy crawl brandenburg -s "CLOSESPIDER_ITEMCOUNT=3"
+scrapy crawl niedersachsen -s "CLOSESPIDER_ITEMCOUNT=3"
```
Comment (cyroxx) [inline scrape_all.sh:2]: You only included Brandenburg and Niedersachsen. Is there anything that keeps us from expanding this to the other states?
Also, what are you trying to achieve with `-s "CLOSESPIDER_ITEMCOUNT=3"`?

**Review by cyroxx — COMMENTED** [review]
Could you add some documentation on how to use the resulting Docker container?
An aspect that might be included:

- what needs to be done to properly setup the PostgreSQL database and use it from the container?

Apart from that: There is a crontab - so I guess you are trying to regularly run the scrapers from within the Docker container? If that is the case: scheduling the regular runs from the outside (i. e. the Docker host) would be the preferable option, I think.

File: scrape_all.sh (hunk 1 above)
Comment (k-nut) [reply scrape_all.sh]: Fixed in 3bb2795

**Comment by k-nut** [comment]
I have decided against using a crontab at all for now. I think that at this stage it makes sense to trigger the runs manually when we want to get the newest data.

**Comment by k-nut** [comment]
@cyroxx documentation is updated. Please take another look.

**Review by cyroxx — APPROVED** [review]
LGTM

**Pull request merged**